ci/install_linux.sh

```diff
@@ -12,3 +12,6 @@ cd build
 cmake ..
 make
 sudo make install
+
+# Code-coverage
+sudo apt-get -qq --yes --force-yes install lcov curl
```

Comment:
Maybe delete this and add lcov and curl to the packages list in `travis.yml` (assuming that lcov and curl have been whitelisted by Travis CI)? This would allow them to be cached by Travis.